core/trino-spi/src/main/java/io/trino/spi/predicate/Marker.java

```diff
@@ -40,8 +40,9 @@
  * A point on the continuous space defined by the specified type.
  * Each point may be just below, exact, or just above the specified value according to the Bound.
  */
+// TODO remove
```

Comment:
Add a deprecation notice and what to use instead

Reply:
There is no direct drop-in replacement, and the class is already `@Deprecated`.

If you don't think this is a good idea, i can remove it completely, without "soft".

Reply:
AFAIK the replacement is to use domains explicitly?